app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModel.kt

```diff
@@ -298,12 +298,26 @@ class GroupConversationDetailsViewModel @Inject constructor(
         enableGuestAndNonTeamMember: Boolean,
         enableServices: Boolean,
         conversationId: ConversationId
-    ) = updateConversationAccessRole(
-        allowGuest = enableGuestAndNonTeamMember,
-        allowNonTeamMember = enableGuestAndNonTeamMember,
-        allowServices = enableServices,
-        conversationId = conversationId
-    )
+    ): UpdateConversationAccessRoleUseCase.Result {
+
+        val accessRoles = Conversation
+            .accessRolesFor(
+                guestAllowed = enableGuestAndNonTeamMember,
+                servicesAllowed = enableServices,
+                nonTeamMembersAllowed = enableGuestAndNonTeamMember
+            )
+
+        val access = Conversation
+            .accessFor(
+                guestsAllowed = enableGuestAndNonTeamMember
+            )
```

Comment:
Can we make it a one liner for better readability?

Reply:
I find this to be more readable than 1 line